scripts/umbrel-os/status-server/static/index.html

```diff
@@ -67,28 +67,28 @@
       </svg>
 
       <div class="error-text monitor-check">
-        <p><b class="text-dark">Error: External drive not connected</b></p>
-        <p>This can sometimes be caused by using an unofficial power supply.</p>
+        <p><b class="text-dark">Error: External drive disconnected</b></p>
+        <p>The external drive was disconnected while Umbrel was running. This can sometimes happen when using an unofficial Raspberry Pi power supply.</p>
```

Comment:
Why the Raspberry Pi distinction here? It could happen on many SBCs.

Reply:
But the status server only runs on Umbrel OS, which runs only Raspberry Pi 4?